<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeptonAndCamera</class>
 <widget class="QMainWindow" name="LeptonAndCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LeptonAndCamera</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="leptonview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Cameraview">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>48</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Lepton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>48</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pic">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeCam">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CloseCam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>85</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OpenCamera</string>
    </property>
   </widget>
   <widget class="QLabel" name="leptonView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CameraView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
